src/unity/python/README.md

```diff
@@ -1,132 +0,0 @@
-* [Recommender systems](https://apple.github.io/turicreate/docs/userguide/recommender/introduction.html)
-* [Image classification](https://apple.github.io/turicreate/docs/userguide/image_classifier/introduction.html)
-* [Image similarity](https://apple.github.io/turicreate/docs/userguide/image_similarity/introduction.html)
-* [Object detection](https://apple.github.io/turicreate/docs/userguide/object_detection/introduction.html)
-* [Activity classifier](https://apple.github.io/turicreate/docs/userguide/activity_classifier/introduction.html)
-* [Text classifier](https://apple.github.io/turicreate/docs/userguide/text_classifier/introduction.html)
-
-You can also work with essential machine learning models, organized into algorithm-based toolkits:
-* [Classifiers](https://apple.github.io/turicreate/docs/userguide/supervised-learning/classifier.html)
-* [Regression](https://apple.github.io/turicreate/docs/userguide/supervised-learning/regression.html)
-* [Graph analytics](https://apple.github.io/turicreate/docs/userguide/graph_analytics/intro.html)
-* [Clustering](https://apple.github.io/turicreate/docs/userguide/clustering/intro.html)
-* [Nearest Neighbors](https://apple.github.io/turicreate/docs/userguide/nearest_neighbors/nearest_neighbors.html)
-* [Topic models](https://apple.github.io/turicreate/docs/userguide/text/intro.html)
-
-System Requirements
--------------------
-
-* Python 2.7 (Python 3.5+ support coming soon)
-* x86\_64 architecture
-* macOS 10.11+, Linux with glibc 2.12+ (including WSL on Windows 10)
-
-Installation
-------------
-
-For detailed instructions for different varieties of Linux see [LINUX\_INSTALL.md](https://github.com/apple/turicreate/LINUX_INSTALL.md).
-For common installation issues see [INSTALL\_ISSUES.md](https://github.com/apple/turicreate/INSTALL_ISSUES.md).
-
-We recommend using virtualenv to use, install, or build Turi Create. 
-Be sure to install virtualenv using your system pip.
-
-```shell
-pip install virtualenv
-```
-
-The method for installing *Turi Create* follows the
-[standard python package installation steps](https://packaging.python.org/installing/).
-To create a Python virtual environment called `venv` follow these steps:
-
-```shell
-# Create a Python virtual environment
-cd ~
-virtualenv venv
-```
-
-To activate your new virtual environment and install `Turi Create` in this environment, follow these steps:
-```shell
-# Active your virtual environment
-source ~/venv/bin/activate
-
-# Install Turi Create in the new virtual environment, pythonenv
-(venv) pip install -U turicreate
-```
-
-Documentation
--------------
-
-The package [User Guide](https://apple.github.io/turicreate/docs/userguide) and [API Docs](https://apple.github.io/turicreate/docs/api) contain
-more details on how to use Turi Create.
-
-GPU Support
------------
-
-By default, `turicreate` takes a dependency on the default installation of
-`mxnet`. To enable GPU support after installation of the `turicreate` package,
```

Comment:
This section is not required in the pypi readme.

Reply:
Why not?